### PR TITLE
extend entrypoint to allow ralphctl

### PR DESCRIPTION
**Pull request by szymi-**

### CODE REVIEW

File: docker/provision/docker-entrypoint.sh [hunk 1]
```diff
@@ -51,6 +52,9 @@ case "$ARG" in
     upgrade)
         "${RALPH_LOCAL_DIR}/init-ralph.sh" upgrade
         ;;
+    ralphctl)
```
Comment (romcheg) [inline docker/provision/docker-entrypoint.sh:55]: Isn't it goint to do the same thing as 

`docker run --entrypoint ralphctl <container> <extra_args>
`
?

Comment (szymi-) [reply docker/provision/docker-entrypoint.sh:55]: Yes, but before that it will load config from the config files.

Comment (romcheg) [reply docker/provision/docker-entrypoint.sh:55]: The line above will also read configuration files -- that's what ralphctl does.

Comment (szymi-) [reply docker/provision/docker-entrypoint.sh:55]: I need the stuff that happens from line 40 of this file to have db connection configured for instance.

Comment (romcheg) [reply docker/provision/docker-entrypoint.sh:55]: ralphctl does that on its own: https://github.com/allegro/ralph/blob/63316f4d9938c42cebe88ebdc1210a829b4b1dae/contrib/common/usr/bin/ralphctl#L5

Comment (szymi-) [reply docker/provision/docker-entrypoint.sh:55]: Ok, then this is uselss.

**Pull request closed without merging**